maven/settings-security/invalid/not-base64.xml

```diff
@@ -0,0 +1,3 @@
+<settingsSecurity>
```

Comment:
Could we put these in places like

```
maven/{valid,invalid}/settings-security-not-base64.xml
```

That way it's:

`tool-or-service/validity/single-test-case(\.extension)?`

where single test-case can be a folder or file depending on if you need to build out a folder structure for the path based matching to work.

Reply:
Done.